src/components/VoteBox.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import styled from "styled-components";
+export default function VoteBox() {
+  const [voteDataList, setVoteDataList] = useState([]);
+
+  const sortVoteDataList = (dataList) => {
+    const returnDataList = dataList
+      .concat()
+      .sort((a, b) => (a.voteCount > b.voteCount ? -1 : 1));
+    return returnDataList;
+  };
+
+  useEffect(() => {
+    const fetchVoteData = async () => {
+      try {
+        const response = await axios.get(
+          "http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/candidates"
+        );
+        const initialVoteDataList = sortVoteDataList(response.data);
+        setVoteDataList(initialVoteDataList);
+      } catch (e) {
+        console.log(e);
+      }
+    };
+    fetchVoteData();
+  }, []);
```

Comment:
sort를위한 함수를 따로빼두신 것도 좋은데, 이렇게 간단하게도 할 수 있을 것 같아요 :)
그리고 sort함수는 원본배열을 정렬하기때문에 따로 변수선언해서 넣어주실 필요 없습니당 [참고](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/sort)


```
 const sortVoteDataList = (dataList) => {
    return dataList.sort((a, b) => b.voteCount - a.voteCount);
  };
```


```suggestion

  useEffect(() => {
  const fetchVoteData = async () => {
      try {
        const {data}= await axios.get(
          "http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/candidates"
        );
        data.sort((a, b) => b.voteCount - a.voteCount)
        setVoteDataList(data);
      } catch (e) {
        console.log(e);
      }
    };
    fetchVoteData();
  }, []);
```

Reply:
와 ... 이렇게 간결해질 수 있다뇨 ㅠㅠㅠ 감동입니다...
response가 아닌 response.data를 바로 가져다 쓰는 것이 참 효율적입니다..! sort 함수가 원본 배열을 변화시킨다는 것도 처음 알았네요..! 감사합니다!!